Tangem/App/Services/Common/Analytics/Analytics.swift

```diff
@@ -16,13 +16,12 @@ import TangemSdk
 
 enum Analytics {
     static private var analyticsSystems: [Analytics.AnalyticSystem] = [.firebase, .appsflyer, .amplitude]
-    static private var persistentParams: [ParameterKey: String] = [:]
-
-    static func reset() {
-        persistentParams = [:]
-    }
 
     static func log(_ event: Event, params: [ParameterKey: String] = [:]) {
+        guard !AppEnvironment.current.isXcodePreview else {
```

Comment:
лучше без отрицания)
```suggestion
        if AppEnvironment.current.isXcodePreview {
```